Web/controllers/dashboard.js

```diff
@@ -1512,7 +1512,7 @@ controllers.other.nameDisplay = async (req, res) => {
 		exampleUsername: req.consolemember.user.username,
 		exampleNickname: req.consolemember.nickname,
 		exampleDiscriminator: req.consolemember.user.discriminator,
-		currentExample: client.getName(svr, serverDocument, req.consolemember),
+		currentExample: client.getName(serverDocument, req.consolemember),
```

Comment:
Web/controllers/dashboard.js no longer exists, see 3f4bbaeaae141823940c841cd049ececf858b48c

Reply:
Oh well, time for my non-existent git skills to shine.